README.rst

```diff
@@ -1,17 +1,18 @@
 ==================================
-XBlock Flow Control |build-status|
+XBlock Flow Control
 ==================================
+|build-status| 
+|PyPI| 
+|PyPI license| 
 
 The Flow Control XBlock provides a way to display the content of a unit or to redirect the user elsewhere based on compliance with a condition that evaluates the submission or the score of a problem or a set of problems.
 
 Installing on Open edX Devstack
 -------------------------------
 
-Inside both LMS and Studio shells, using `make lms-shell` and `make studio-shell` in your devstack directory:
-* Clone this repository
-* Inside the newly downloaded directory, perform::
+Inside both LMS and Studio shells, using `make lms-shell` and `make studio-shell` in your devstack directory do:
 
-    make install
+    pip install flow-control-xblock
```

Comment:
I think that this is the recommended way to install on servers, but for the development it is better to install this cloning the repo and then installing this.

Reply:
Updated the README instructions